backend/src/app/features/components_v2.clj

```diff
@@ -995,6 +996,9 @@
   [assets generic-name]
   (let [;; Group by first element of the path.
         groups (d/group-by #(first (cfh/split-path (:path %))) assets)
+        ;; If there is a group called as the generic-name we have to preserve it
+        unames (into #{} (keep str) (keys groups))
+        groups (rename-keys groups {generic-name (cfh/generate-unique-name unames generic-name)})
```

Comment:
This may cause conflicts if there is an existing group with many components inside, that is split by the code below

Reply:
It works :)
![3](https://github.com/penpot/penpot/assets/1579633/7ada7ba0-a70a-4f4a-9447-f0d818afecf2)